chirp/drivers/tdh8.py

```diff
@@ -2340,8 +2337,10 @@ class TDH3_GMRS(TDH3):
     MODEL = "TD-H3-GMRS"
     ident_mode = b'P31184\xff\xff'
     _gmrs = True
-    _vhf_range = (136000000, 175000000)
-    _uhf_range = (400000000, 521000000)
+    _bands = [(50000000, 76000000), (76000000, 108000000),
+              (108000000, 136000000), (136000000, 174000000),
+              (174000000, 350000000), (350000000, 400000000),
+              (400000000, 470000000), (470000000, 600000000)]
```

Comment:
Okay, but if you look at the issue attached to my attempt, this didn't work for the reporter. I'm not sure if it didn't work because of the used flags thing, or because there is more required to be done in the driver, which is what I was hoping to get feedback on. You don't have an H3 so you can't actually test that this works right?

Reply:
Correct, I cannot test. 

I couldn't find anything that references the _uhf_range etc (now _bands) either. So I assumed by updating all of those instances it would have the correct bands available. If that doesn't work, it might be hard-coded somewhere. I don't have a lot of time lately, hoped this would work/move the right direction at least

Reply:
Okay I missed the request for test above, so I see now.

@songjunbiao will you confirm if these band expansions work properly?

Reply:
I've taken his advice and merged the codes, so far only the H3's work, the others don't, and I've wrapped H3 GMRS and H3 Ham.